src/redux/books/booksSlice.js

```diff
@@ -1,4 +1,6 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { API_URL } from '../../helper/config';
 
 const initialState = {
```

Comment:
- [ ] I think initiating a fixed value for your book is making the app buggy because when this is deleted and the page gets refreshed, it shows up again. Can you try to fix this?

![mobile](https://github.com/Yonashenok/Bookstore-App/assets/99715640/57b3ac03-9348-4d25-bc40-49172901bcd9)